lib/global_stats.rb

```diff
@@ -1,13 +1,86 @@
+require 'gds_api/rummager'
+
 class GlobalStats
   include StatsHelpers
 
+  BLACKLIST_DOCUMENT_TYPES = %w[
```

Comment:
Out of interest, where did this list of document types come from? Just wondering about managing this list across different apps... maybe in future we might want a new document type that store this information...? 😐

Reply:
Yes we were considering creating a supertype for this